frontend/src/components/program/DialogActivityCreate.vue

```diff
@@ -61,15 +198,94 @@ export default {
               deleted: false,
             },
           ],
+          copyActivitySource: null,
         })
       } else {
+        this.canReadClipboard = 'unknown'
         // clear the variable parts of the form on exit
+        this.copyActivitySource = null
+        this.copyActivitySourceUrl = null
         this.entityData.location = ''
         this.entityData.scheduleEntries = []
       }
     },
+    copyActivitySourceUrl: function (url) {
+      this.copyActivitySourceUrlLoading = true
+
+      this.getCopyActivitySource(url).then(
+        (activityProxy) => {
+          if (activityProxy != null) {
+            activityProxy._meta.load.then(
+              (activity) => {
+                this.$set(this, 'copyActivitySource', activity)
+                this.$set(this, 'copyContent', activity != null)
+                this.copyActivitySourceUrlLoading = false
+              },
+              () => {
+                this.copyActivitySourceUrlLoading = false
+              }
+            )
+          } else {
+            this.$set(this, 'copyActivitySource', null)
+            this.$set(this, 'copyContent', false)
+            this.copyActivitySourceUrlLoading = false
+          }
+
+          // if Paste-Popover is shown, close it now
+          if (this.copyActivitySourceUrlShowPopover) {
+            this.$nextTick(() => {
+              this.$set(this, 'copyActivitySourceUrlShowPopover', false)
+            })
+          }
+        },
+        () => {
+          this.copyActivitySourceUrlLoading = false
+        }
+      )
+    },
   },
   methods: {
+    refreshCopyActivitySource() {
+      navigator.permissions.query({ name: 'clipboard-read' }).then(
```

Comment:
minor:
On mobile duckduckgo browser (and probably firefox) this leads to an error.
https://ecamp.sentry.io/issues/4633431420
Maybe change the onRejected handler to a catch which catches the whole chain?

Reply:
I think you have to add a `.catch()` after the `.then(…)`.